app/forms/hyrax/hyrax_form.rb

```diff
@@ -11,8 +13,6 @@ def notable_rights_statement?
     def self.multiple?(field)
```

Comment:
OK, that's fine — I've created an issue in Grocer for this: https://github.com/pulibrary/grocer/issues/24